pyaadhaar.egg-info/SOURCES.txt

```diff
@@ -1,6 +1,7 @@
 setup.py
 pyaadhaar/__init__.py
 pyaadhaar/decode.py
+pyaadhaar/tests.py
```

Comment:
fix the tests.py with proper testcase or remove

Reply:
Sure! will add this